### DISCUSSION TITLE
使用cloudflare中转后下载文件报错

### DISCUSSION THREAD

**Discussion by FmgX** [AlistGo/alist#1008] [Q&A] [answered]
没有挂端口号，有协议头
![](https://tu.121318.xyz/杂/202204281746128.png)
![](https://tu.121318.xyz/杂/202204281748780.png)
![](https://tu.121318.xyz/杂/202204281748166.png)
下载时出现1101错误
直接访问cf的连接打开是百度
token从这里找的
![](https://tu.121318.xyz/杂/202204281752046.png)

求大佬

**Comment by FmgX** [comment]
求大佬

**Comment by xhofe** [comment, marked as answer]
你这个地址我都打不开。

**Reply by FmgX** [reply]
cs.fongsera.online

**Reply by FmgX** [reply]
打不开网站吗

**Reply by xhofe** [reply]
你似乎禁止了某些地区的ip

**Reply by xhofe** [reply]
国内可以打开，国外的全部无法访问。

**Reply by FmgX** [reply]
奥那个我也不知道怎么回事，香港的机子，挂代理打不开。clash正好也能分流也就没管过这个

**Reply by xhofe** [reply]
cf肯定也访问不了你的地址了，所以报错。

**Reply by FmgX** [reply]
我换个国内的试试

**Reply by FmgX** [reply]
解决了，原因在于Nginx防火墙，早期安装好了没有理会，里面有禁止了海外IP访问
